services/api-server/tests/unit/test_api_wallets.py

```diff
@@ -1,5 +1,6 @@
+from collections.abc import Callable
 from pathlib import Path
-from typing import Any, Callable
+from typing import Any
```

Comment:
Just out of curiosity, why do you prefer to import `Callable` from `collections.abc` instead of `typing`?